src/nanovg.c

```diff
@@ -2119,22 +2119,30 @@ void nvgRect(NVGcontext* ctx, float x, float y, float w, float h)
 
 void nvgRoundedRect(NVGcontext* ctx, float x, float y, float w, float h, float r)
 {
-	if (r < 0.1f) {
-		nvgRect(ctx, x,y,w,h);
+	nvgRoundedRectEx(ctx, x, y, w, h, r, r, r, r);
+}
+
+void nvgRoundedRectEx(NVGcontext* ctx, float x, float y, float w, float h, float r1, float r2, float r3, float r4)
```

Comment:
I suppose in the case of the round corners, I wonder if you would ever want a separate radius for each of the corners.